040_More_Documentation.html.md

```diff
@@ -13,5 +13,33 @@ This section describes the details of your solution in a logical order. For exam
 
 #### Example
 For an example of unique solution documentation, see the <a href="https://devportal.yaas.io/solutions/saphybrisprofile/index.html">SAP Hybris Profile</a> documentation.
+
+### README files
+
+README files describe other files in a given directory.
+
+When creating external README files, use this template:
+
+````
+```
+Project Title converted to ASCII art using an ASCII generator. For example, http://patorjk.com/software/taag/#p=display&f=Big&t=Project%20Title%0A. Use the following font: Big.
```

Comment:
Convert the **Project Title** to ASCII art using an ASCII generator, and use the **Big** font. For example, http://patorjk.com/software/taag/#p=display&f=Big&t=Project%20Title%0A.

Need to update this in the SF repo as well.